Without this patch, you see weird things like substitutions that turn
/var/tmp/nstroot/usr/share/man/man1 to /var/tmp/ntsroot/usr/share/man/$(PREFIX),
which is bad.

--- perl-5.6.0/lib/ExtUtils/MM_Unix.pm	Sat Jun 17 16:57:55 2000
+++ perl-5.6.0/lib/ExtUtils/MM_Unix.pm	Sat Jun 17 16:57:43 2000
@@ -3097,6 +3097,7 @@
 
 sub prefixify {
     my($self,$var,$sprefix,$rprefix) = @_;
+    return if (length($sprefix) <= 0);
     $self->{uc $var} ||= $Config{lc $var};
     $self->{uc $var} = VMS::Filespec::unixpath($self->{uc $var}) if $Is_VMS;
     $self->{uc $var} =~ s/\Q$sprefix\E/$rprefix/s;
